bin/run-ldc93s1.sh

```diff
@@ -19,5 +19,6 @@ python -u DeepSpeech.py \
   --dev_batch_size 1 \
   --test_batch_size 1 \
   --n_hidden 494 \
+  --checkpoint_dir ="ds2-v2"
```

Comment:
This is orthogonal to the main thrust of this PR, fixing `FLAGS` parameters, and doesn't adhere to the standard of other scripts[[1](https://github.com/mozilla/DeepSpeech/blob/master/bin/run-ted.sh#L20)][[2](https://github.com/mozilla/DeepSpeech/blob/master/bin/run-librivox.sh#L20)][[3](https://github.com/mozilla/DeepSpeech/blob/master/bin/run-fisher.sh#L20)]... So this should be removed or made to adhere to the existing standard.